### PR TITLE
Update the payment type to allow easier removal and switching

### PR DESCRIPTION
**Pull request by liamkeaton**

### CODE REVIEW

File: styles/payment-type.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	&__payment-type {
+		grid-row: 1;
+
+		img {
+			height: 15px;
+		}
+
+		label {
+			height: 15px;
+			width: calc(100% - 5px);;
```
Comment (jkerr321) [inline styles/payment-type.scss]: unnecessary `;`

File: utils/payment-type.js [hunk 2]
```diff
@@ -23,6 +25,15 @@ class PaymentType {
 
 		this.element = element;
 		this.$paymentType = element.querySelector('.ncf #paymentTypeField');
+		this.$directDebitGuarantee = this.$paymentType.querySelector('.ncf #directDebitGuarantee');
```
Comment (jkerr321) [inline utils/payment-type.js]: What's the logic / convention behind putting `$` before some variable names?

Comment (liamkeaton) [reply utils/payment-type.js]: @remybach does this to indicate a DomElement

Comment (jkerr321) [reply utils/payment-type.js]: oooooooooo ok cool

File: styles/payment-type.scss (hunk 1 above)
Comment (liamkeaton) [reply styles/payment-type.scss]:
```suggestion
			width: calc(100% - 5px);
```

**Pull request merged**